Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1103,8 +1103,8 @@ def define_Configs(self):
             self.VALIDATIONDefaultCFF="Configuration/StandardSequences/ValidationHeavyIons_cff"
             self.VALIDATIONDefaultSeq=''
             self.EVTCONTDefaultCFF="Configuration/EventContent/EventContentHeavyIons_cff"
-            self.RECODefaultCFF="Configuration/StandardSequences/ReconstructionHeavyIons_cff"
-            self.RECODefaultSeq='reconstructionHeavyIons'
+            self.RECODefaultCFF="Configuration/StandardSequences/Reconstruction_cff"
+            self.RECODefaultSeq='reconstruction'
```

Comment:
Yeah, it would, and thanks for pointing it out.  However, `--scenario HeavyIons` is actually still used in the gen step to specify genJet clustering that is appropriate for generators comprised of multiple "sub-events".  As as starting point, I'm simply removing the old heavy-ion reconstruction, without touching the other things that are modified by this `scenario`.  This takes care of the code which is giving the most maintenance problems.  There are further things that can and should be cleaned up, but it's more practical for me to tackle it in chunks.